code/game/machinery/tcomms/presets.dm

```diff
@@ -31,3 +31,7 @@
 	hidden_link = TRUE
 	password_bypass = TRUE
 
+GLOBAL_VAR(cc_tcomms_relay_uid)
+/obj/machinery/tcomms/relay/cc/Initialize(mapload)
+	. = ..()
+	GLOB.cc_tcomms_relay_uid = UID(src)
```

Comment:
```suggestion
/obj/machinery/tcomms/relay/cc/Initialize(mapload)
	. = ..()
	GLOB.cc_tcomms_relay_uid = UID()
```

UID() is a self-scoped proc not a global that takes a datum as an arg.

Reply:
thanks, missed that one <3